js/client-battle.js

```diff
@@ -571,13 +571,14 @@
 				if (this.choice && this.choice.waiting && !this.finalDecision) {
 					buf += '<button name="undoChoice">Cancel</button>';
 				}
-				buf += '</p>';
+				buf += '</p><p class="timer">';
```

Comment:
To clarify, by "new Google thing we saw", I meant the V8 optimization thing: https://github.com/petkaantonov/bluebird/wiki/Optimization-killers